diff --git a/components/mpas-seaice/src/column/ice_kinds_mod.F90 b/components/mpas-seaice/src/column/ice_kinds_mod.F90
index 4b12177c78..9f17c90729 100644
--- a/components/mpas-seaice/src/column/ice_kinds_mod.F90
+++ b/components/mpas-seaice/src/column/ice_kinds_mod.F90
@@ -20,7 +20,8 @@
                             log_kind  = kind(.true.), &
                             int_kind  = selected_int_kind(6), &
                             real_kind = selected_real_kind(6), &
-                            dbl_kind  = selected_real_kind(13), &
+                            dbl_kind  = selected_real_kind(6), &
+
                             r16_kind  = selected_real_kind(26)

 !=======================================================================
